packages/zendesk-adapter/src/change_validators/utils.ts

```diff
@@ -13,5 +13,30 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
+import Joi from 'joi'
+import { InstanceElement, ReferenceExpression } from '@salto-io/adapter-api'
+import { createSchemeGuardForInstance } from '@salto-io/adapter-utils'
+import { ARTICLES_FIELD, CATEGORIES_FIELD, SECTIONS_FIELD } from '../constants'
+
 export const createEmptyFieldErrorMessage = (fullName: string, fieldName: string): string =>
   `Can not change ${fullName}' ${fieldName} to be empty`
+
+type articlesOrderType = InstanceElement & { value: { articles: ReferenceExpression[] } }
+type sectionsOrderType = InstanceElement & { value: { sections: ReferenceExpression[] } }
+type categoriesOrderType = InstanceElement & { value: { categories: ReferenceExpression[] } }
```

Comment:
please start types with capital letters
```suggestion
type ArticlesOrderType = InstanceElement & { value: { articles: ReferenceExpression[] } }
type SectionsOrderType = InstanceElement & { value: { sections: ReferenceExpression[] } }
type CategoriesOrderType = InstanceElement & { value: { categories: ReferenceExpression[] } }
```